src/services/bleManager/ble-manager.service.ts

```diff
@@ -121,50 +119,31 @@ export class BleManagerService {
         this.uiLog.log('Scales found ' + JSON.stringify(devices));
         resolve(devices);
       };
-      ble.startScan([], async (device)=>  {
+
+      ble.startScan([], async (device) => {
         devices.push(device);
-        let deviceName = '';
-        if (device && device.name && device.name !== '') {
-          deviceName = device && device.name && device.name.toLowerCase();
-        }
-        if (deviceName.startsWith('decent')) {
+        if (DecentScale.test(device) || LunarScale.test(device)) {
```

Comment:
What would happen here, if decent scale and lunar scale is one?
The first one would win right?

Reply:
Yeah, probably decent will come up first. I don't think this is a problem now, but maybe in the future, it will be nice to have a selection if there are multiple devices detected. The same issue can happen if you have few decent or lunar scales turned on for some reason.